packages/devtools_app/lib/src/analytics/constants.dart

```diff
@@ -158,47 +139,54 @@ String topicDocumentationButton(String topic) => '${topic}DocumentationButton';
 String topicDocumentationLink(String topic) => '${topic}DocumentationLink';
 
 /// Analytic time constants specific for memory screen.
-class MemoryTimeAnalytics {
+class MemoryTime {
   static const adaptSnapshot = 'adaptSnapshot';
+  static const calculateDiff = 'calculateDiff';
   static const updateValues = 'updateValues';
 }
 
 /// Analytic event constants specific for memory screen.
-class MemoryEventAnalytics {
+class MemoryEvent {
   static const gc = 'gc';
   static const settings = 'settings';
+  static const autoSnapshot = 'autoSnapshot';
 
+  static const chartLegend = 'chartLegend';
+  static const chartAndroid = 'chartAndroid';
   static const chartExpand = 'chartExpand';
   static const chartCollapse = 'chartCollapse';
+  static const chartInterval = 'chartInterval';
 
   static const profileDownloadCsv = 'profileDownloadCsv';
   static const profileRefreshManual = 'profileRefreshManual';
   static const profileRefreshOnGc = 'profileRefreshOnGc';
+  static const profileHelp = 'memoryProfileHelp';
 
-  static const allocationClear = 'allocationClear';
-  static const allocationRefresh = 'allocationRefresh';
-  static const allocationFilter = 'allocationFilter';
-  static const allocationTrace = 'allocationTrace';
-  static const allocationHelp = 'allocationHelp';
+  static const tracingClear = 'tracingClear';
+  static const tracingRefresh = 'tracingRefresh';
+  static const tracingClassFilter = 'tracingClassFilter';
+  static const tracingTraceCheck = 'tracingTraceCheck';
+  static const tracingHelp = 'memoryTracingHelp';
 
-  static const diffTakeSnapshot = 'diffTakeSnapshot';
+  static const diffTakeSnapshotControlPane = 'diffTakeSnapshotControlPane';
+  static const diffTakeSnapshotAfterHelp = 'diffTakeSnapshotAfterHelp';
   static const diffClearSnapshots = 'diffClearSnapshots';
 
-  static const diffSnapshotDiff = 'diffSnapshotDiff';
+  static const diffSnapshotDiffSelect = 'diffSnapshotDiffSelect';
+  static const diffSnapshotDiffOff = 'diffSnapshotDiffSelectOff';
   static const diffSnapshotFilter = 'diffSnapshotFilter';
   static const diffSnapshotDownloadCsv = 'diffSnapshotDownloadCsv';
   static const diffSnapshotDelete = 'diffSnapshotDelete';
 
-  static const diffClassSelect = 'diffClassSelect';
+  static const diffClassDiffSelect = 'diffClassDiffSelect';
+  static const diffClassSingleSelect = 'diffClassSingleSelect';
   static const diffPathSelect = 'diffPathSelect';
+  static const diffClassDiffCopy = 'diffClassDiffCopy';
+  static const diffClassSingleCopy = 'diffClassSingleCopy';
   static const diffPathCopy = 'diffPathCopy';
   static const diffPathFilter = 'diffPathFilter';
-  static const diffPathUnfilter = 'diffPathUnfilter';
-  static const diffPathRevert = 'diffPathRevert';
-  static const diffPathUnrevert = 'diffPathUnrevert';
+  static const diffPathInvert = 'diffPathRevert';
```

Comment:
change string to match var name 'diffPathInvert' (changing this string is okay since the feature sending this event hasn't been released yet, so we don't have existing data using 'diffPathRevert')